controllers/api/v1alpha1/cfprocess_types.go

```diff
@@ -39,6 +39,9 @@ type CFProcessSpec struct {
 	// Command string used to run this process on the app image. This is analogous to command in k8s and ENTRYPOINT in Docker
 	Command string `json:"command,omitempty"`
 
+	// The default command for this process as defined by the build. This field is ignored when the Command field is set
+	DropletCommand string `json:"dropletCommand,omitempty"`
```

Comment:
We're discussing the naming of this field. It gets pulled from the OCI image config labels, so we're not quite sure what to name it. In a sense, it is the default command, but default is a term we want to be pretty careful about using. Anyone have additional thoughts?

Reply:
How about we call this field `Command` and instead rename the old `Command` to `CustomCommand`, or `UserSpecifiedCommand` or something like this? I guess that such a change would be even transparent to currently running workloads, i.e. hopefully there would be no upgrade issues.

Reply:
I think it's useful to leave the user-specific command field as just `command` since that aligns with the existing CF APIs.

https://v3-apidocs.cloudfoundry.org/version/3.128.0/index.html#the-process-object

![Screen Shot 2022-11-14 at 10 14 38 AM](https://user-images.githubusercontent.com/2096955/201723634-aa7accf1-75d8-4a83-b72a-a349ca8a3f55.png)

I'm not sure if it's publicly referred to as this often, we've referred to the "droplet command" as 
the "detected command" because the buildpacks use that that terminology a lot (I think in CF for VMs it comes from a callback object called the "buildpack detection receipt" for example).

Here's some various naming there.
https://github.com/cloudfoundry/cloud_controller_ng/blob/3dcc660170c86c581b0d26ee4290028f38cbbd75/app/models/runtime/process_model.rb#L410-L412

So I have a slight preference to `DetectedCommand` or `ImageCommand` or something. `dropletCommand` is fine too, I'm just not sure how much life the "droplet" metaphor has left in it on Korifi.

I'm also not entirely opposed to `UserSpecifiedCommand` like @danail-branekov suggested since it's clear, but IMO there is value to being able to say this `CFProcess` looks very close to "V3 CF API Process" and that `Command` means the same thing in both contexts.

Reply:
Thanks all, we went with `DetectedCommand` just to expunge the droplet metaphor